html/janus.js

```diff
@@ -1915,7 +1915,7 @@ function Janus(gatewayCallbacks) {
 			try {
 				pluginHandle.onremotetrack(event.track, mid, true);
 			} catch(e) {
-				Janus.error(e);
+				Janus.error("Error calling track event handler", e);
```

Comment:
I'd say `onremotetrack` instead of `track event handler`, since we have two different handlers for tracks, depending on whether they're local or remote. Same for the lines you changed below.